pyiron/base/job/generic.py

```diff
@@ -1131,6 +1183,8 @@ def _run_if_collect(self):
         status is set to 'finished'
         """
         self.collect_output()
+        self.project_hdf5.rewrite_hdf5(job_name=self.job_name, exclude_groups=self._exclude_groups_hdf,
```

Comment:
This is the critical part because it affects not only individual calculations like `VASP` or `Lammps` but also `SerialMasters` and `ParallelMasters`.